examples/messaging/messaging/client.py

```diff
@@ -1,36 +1,39 @@
 import threading
-import urllib2
+import requests
+from requests import Request, Session
+from requests.auth import HTTPBasicAuth
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from requests import RequestException
+
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
 import json
 import time
 import curses
 
-
 _SERVER = 'http://localhost:6543'
 
-
 def post(message, token):
-    headers = {'X-Messaging-Token': token}
-    req = urllib2.Request(_SERVER, headers=headers)
-    req.get_method = lambda: 'POST'
-    message = {'text': message}
-    req.add_data(json.dumps(message))
-    urllib2.urlopen(req)
-
+    data = {'text': message}
+    s = Session()
+    req = Request('POST', _SERVER, json=data)
+    prepped = s.prepare_request(req)
+    prepped.headers['X-Messaging-Token'] = token
+    resp = s.send(prepped)
```

Comment:
Why not just `requests.post(_SERVER, data=data, headers={'X-Messaging-Token': token})` ?